molgenis/molgenis-emx2
# feat(tw): init listbox component

What are the main changes you did:

- [x] created a listbox component as an alternative to select component (see #4446).
- [x] added e2e tests for the component
- [x] added story
- [x] integrated component into form components

how to test:
- navigate to the [story](https://preview-emx2-pr-4421.dev.molgenis.org/apps/tailwind-components/#/input/Listbox.story)
- interact with the component using the mouse and keyboard (see issue #4446 for a lists of keyboard events)


todo:
- [ ] Add support for Objects, booleans, and numbers as input.
